deployment/hasura/migrations/Aerie/3_sequence_parcel/down.sql

```diff
@@ -0,0 +1,203 @@
+
+  drop constraint expansion_rule_unique_name_per_parcel_and_model,
+  drop constraint expansion_rule_parcel_id_fkey;
+
+drop view sequencing.expansion_set_rule_view;
+
+create or replace view sequencing.expansion_set_rule_view as
+select str.set_id,
+       rule.id,
+       rule.activity_type,
+       rule.expansion_logic,
+       rule.authoring_command_dict_id,
+       rule.authoring_mission_model_id,
+       rule.created_at,
+       rule.updated_at,
+       rule.name,
+       rule.owner,
+       rule.updated_by,
+       rule.description
+from sequencing.expansion_set_to_rule str
+       left join sequencing.expansion_rule rule
+                 on str.rule_id = rule.id;
+
+alter table sequencing.expansion_rule
+  drop column parcel_id;
+
+
+--------------------------------------
+---- Modify the Expansion Set Table
+--------------------------------------
+alter table sequencing.expansion_set
+  add column command_dict_id integer;
+
+comment on column sequencing.expansion_set.command_dict_id is e''
+  'The ID of a command dictionary';
+
+--- Data Migration
+update sequencing.expansion_set
+set command_dict_id = p.command_dictionary_id
+from sequencing.expansion_set es
+       inner join sequencing.parcel p on es.parcel_id = p.id;
+
+---
+
+alter table sequencing.expansion_set
+  alter column command_dict_id set not null;
+
+alter table sequencing.expansion_set
+  add constraint expansion_set_unique_name_per_dict_and_model
+    unique (mission_model_id, command_dict_id, name),
+
+  add foreign key (command_dict_id)
+    references sequencing.command_dictionary (id)
+    on delete cascade,
+
+  drop constraint expansion_set_unique_name_per_parcel_and_model,
+  drop constraint expansion_set_parcel_id_fkey;
+
+drop view sequencing.rule_expansion_set_view;
+
+create or replace  view sequencing.rule_expansion_set_view as
+select str.rule_id,
+       set.id,
+       set.name,
+       set.owner,
+       set.description,
+       set.command_dict_id,
+       set.mission_model_id,
+       set.created_at,
+       set.updated_at,
+       set.updated_by
+from sequencing.expansion_set_to_rule str left join sequencing.expansion_set set
+                                                    on str.set_id = set.id;
+
+
+alter table sequencing.expansion_set
+  drop column parcel_id;
+
+--------------------------------------
+---- Modify the User Sequence Table
+--------------------------------------
+alter table sequencing.user_sequence
+  add column authoring_command_dict_id integer,
+
+  add foreign key (authoring_command_dict_id)
+    references sequencing.command_dictionary
+    on delete cascade;
+
+
+comment on column sequencing.user_sequence.authoring_command_dict_id is e''
+  'Command dictionary the user sequence was created with.';
+
+--- Data Migration
+update sequencing.user_sequence
+set authoring_command_dict_id = p.command_dictionary_id
+from sequencing.user_sequence us
+       inner join sequencing.parcel p on us.parcel_id = p.id;
```

Comment:
This statement can be reworked to remove the `join` similar to its `up` counterpart:

```sql
update sequencing.user_sequence us
set authoring_command_dict_id = p.command_dictionary_id
from sequencing.parcel p 
where us.parcel_id = p.id;
```

Reply:
omg I am so sorry, I replaced it in the `up.sql` but forgot to replace the `down`. Thank you for caughting that.